server/utils/queries.js

```diff
@@ -25,7 +25,60 @@ const signupUser = async (data) => {
   return user;
 };
 
+const canCreateUpdateAnnouncements = async (id) => {
+  const user = await pool.query('SELECT status FROM users WHERE id=$1', [id]);
+  return user.rows[0].status;
+};
+
+const doesAnnouncementExists = async (title, owner) => {
+  const announcement = await pool.query('SELECT * FROM announcements WHERE title=$1 and owner=$2', [title, owner]);
+  if (announcement.rows.length !== 0) {
+    return true;
+  }
+  return false;
+};
+
+const saveAnnouncement = async (data) => {
+  const announcement = await pool.query('INSERT INTO announcements(title,text,start_date,end_date,status,owner,createdon) VALUES($1,$2,$3,$4,$5,$6,$7) RETURNING*',
+    [
+      data.title,
+      data.description,
+      data.startdate,
+      data.enddate,
+      data.status,
+      data.owner,
+      data.createdon,
```

Comment:
you can just use the spreading operator

Reply:
It's throwing an error when I use spreading